test/Dapr.Client.Test/TestData/InvocationHandlerTestsData.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Xunit;
+
+namespace Dapr.Client.Test.TestData;
+
+public class TryRewriteUriWithNoAppIdRewritesUriToDaprInvokeTestData : TheoryData<string, string, string>
+{
+    public TryRewriteUriWithNoAppIdRewritesUriToDaprInvokeTestData()
+    {
+        Add(null, "http://bank", "https://some.host:3499/v1.0/invoke/bank/method/");
+        Add("bank", "http://bank", "https://some.host:3499/v1.0/invoke/bank/method/");
+        Add("Bank", "http://bank", "https://some.host:3499/v1.0/invoke/Bank/method/");
```

Comment:
If someone could please clarify me the intention behind testing this with a lowercase and uppercase B, that would be amazing, because most of this test data is duplicated and just changes on this casing.

Reply:
It's proving that the operation works in a case-insensitive manner.

Reply:
Just as I suspected. Thanks a lot!